test/controller.bar.tests.js

```diff
@@ -487,4 +489,154 @@ describe('Bar controller tests', function() {
 		expect(bar._model.borderColor).toBe('rgb(0, 255, 0)');
 		expect(bar._model.borderWidth).toBe(1.5);
 	});
+
+	describe('Bar width', function() {
+		beforeEach(function() {
+			// 2 datasets
+			this.data = {
+				labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+				datasets: [{
+					data: [10, 20, 30, 40, 50, 60, 70],
+				}, {
+					data: [10, 20, 30, 40, 50, 60, 70],
+				}]
+			};
+		});
+
+		afterEach(function() {
```

Comment:
@simonbrunel OK!:smiley:

At first, the test results with this PR code merged.
It will succeed.
![jasmine-ok](https://cloud.githubusercontent.com/assets/22541770/20831013/21d270b4-b8be-11e6-95e1-fc36418e1d3b.png)




Next, change the following code.

https://github.com/KoyoSE/Chart.js/blob/e29f2375779e0e2b432e33d8224fba9824551702/src/controllers/controller.bar.js#L154
``` javascritp
// return xScale.options.stacked ? ruler.categoryWidth * xScale.options.barPercentage : ruler.barWidth;
return xScale.options.stacked ? ruler.categoryWidth : ruler.barWidth;
```
Delete `barPercentage` from the bar width of the stacked bar chart. (not horizontalBar chart.)
The bar width becomes wider and the test will fail.
![jasmine-err](https://cloud.githubusercontent.com/assets/22541770/20831023/3059a800-b8be-11e6-8751-a2493508d222.png)
![jasmine-err2](https://cloud.githubusercontent.com/assets/22541770/20831033/3dd29f46-b8be-11e6-8a3a-c927653686b1.png)




console output
```
INFO [framework.browserify]: bundle built
INFO [karma]: Karma v0.12.37 server started at http://localhost:9876/
INFO [launcher]: Starting browser Firefox
INFO [Firefox 50.0.0 (Windows 10 0.0.0)]: Connected on socket nA3rDQIslrjiruLw4C64 with id 47361323
Firefox 50.0.0 (Windows 10 0.0.0): Executed 0 of 348 SUCCESS (0 secs / 0 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 1 of 348 SUCCESS (0 secs / 0.085 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 2 of 348 SUCCESS (0 secs / 0.135 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 3 of 348 SUCCESS (0 secs / 0.211 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 4 of 348 SUCCESS (0 secs / 0.238 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 5 of 348 SUCCESS (0 secs / 0.262 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 6 of 348 SUCCESS (0 secs / 0.332 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 7 of 348 SUCCESS (0 secs / 0.364 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 8 of 348 SUCCESS (0 secs / 0.401 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 9 of 348 SUCCESS (0 secs / 0.438 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 10 of 348 SUCCESS (0 secs / 0.48 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 11 of 348 SUCCESS (0 secs / 0.522 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 12 of 348 SUCCESS (0 secs / 0.546 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 13 of 348 SUCCESS (0 secs / 0.57 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0) Bar controller tests Bar width should correctly set bar width when scale are stacked with min and max options. FAILED
	Expected 379.6 to be close to pixel 341.64000000000004.
	@d:/Users/TOSHIKI/Documents/GitHub/Chart.js/test/controller.bar.tests.js:528:4
Firefox 50.0.0 (Windows 10 0.0.0): Executed 14 of 348 (1 FAILED) (0 secs / 0.6 secs)
Firefox 50.0.0 (Windows 10 0.0.0) Bar controller tests Bar width should correctly set bar width when scale are stacked with min and max options. FAILED
	Expected 379.6 to be close to pixel 341.64000000000004.
	@d:/Users/TOSHIKI/Documents/GitHub/Chart.js/test/controller.bar.tests.js:528:4
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 15 of 348 (1 FAILED) (0 secs / 0.631 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 16 of 348 (1 FAILED) (0 secs / 0.664 secs)
[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 17 of 348 (1 FAILED) (0 secs / 0.722 secs)

(Omitted)

[1AFirefox 50.0.0 (Windows 10 0.0.0): Executed 344 of 348 (1 FAILED) (skipped 4) (7.682 secs / 7.322 secs)
[17:32:09] 'unittest' errored after 29 s
[17:32:09] Error in plugin 'gulp-karma'
karma exited with code 1

```